src/System.Runtime/tests/System/StringTests.cs

```diff
@@ -7,14 +7,15 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Globalization;
+using System.Linq;
 using System.Runtime.InteropServices;
 using System.Threading;
 using System.Threading.Tasks;
 using Xunit;
 
 namespace System.Tests
 {
-    public static class StringTests
+    public static partial class StringTests
```

Comment:
As @justinvp suggests, please do create a new file called StringTests.netcoreapp1.1.cs and move the netcoreapp1.1-specific tests in there. On the .csproj, add this file on the ItemGroup that is conditioned for netcoreapp1.1 only.